### DISCUSSION TITLE
🚀 FR - Warn when changing a select type field's value in the settings.

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3830] [Feature Requests]
### Suggest an idea for ExpressionEngine

So with checkbox, radios, select fields, you specify the values in the field settings.  If at some point you CHANGE the value in the settings, it can cause weird validation issues.  Because any existing saved entries that have the old value are going to trigger a validation error (they aren't in the 'allowed' values any more), but none of the now existing values show up as checked because what was checked is... not one of those values.

So the FR is to give some sort of warning in the field settings when you change a value.  We should at least give a warning when they try to alter an existing value.  Bonus points if it does a check for whether any entries actually have data in that field before issuing the warning.  Extra bonus points if it checks whether the value they're trying to change is actually being used before issuing the warning.  Triple bonus if it gives you a way to alter the old data to match the new value or delete it.

**Comment by intoeetive** [comment]
We have solution for this in https://github.com/ExpressionEngine/ExpressionEngine/pull/3801
